db/seeds.rb

```diff
@@ -1,31 +1,32 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
 
+def translite(str)
+  Russian::translit(str).downcase.gsub(/\W+/, '-')
+end
+
+cities = [["Уфа","ufa"], ["Стерлитамак","sterlitamak"], ["Казань","kazan"], ["Москва","moscow"]]
 City.delete_all
-2.times { City.create(name: Faker::Address.city, uri: Faker::Lorem.word) }
+cities.each { |city| City.create(name: city[0], uri: city[1]) }
 cities = City.all
 
+categories = [
+    ["Авто и мото", ["Продажа", "Покупка", "Сервис", "Услуги", "Другое"]],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.